cql/ManagementLibrary.cql

```diff
@@ -16,15 +16,11 @@ library ManagementLibrary version '1.0.0'
 
 using FHIR version '4.0.1'
 
-include CDSConnectCommonsforFHIRv401 version '1.0.0' called C3F
```

Comment:
I'm curious how you chose which libraries to remove, since there are other included libraries that are also not used in this file?

Reply:
I don't understand the comment. I removed C3F, FHIRHelpers, Common and RiskTableLookup. These were not used in this file. I left Dash, ManageCommonAbnormality, ManageRareAbnormality, ManageSpecialPopulation, Symptomatic and Orders. These libraries are all used in this file.

Reply:
Whoops, I was searching the file by the Library name as opposed to the alias that it's called by. You're right, just the libraries that weren't used in this CQL file were removed.